src/change_stream.ts

```diff
@@ -405,6 +417,8 @@ export class ChangeStreamCursor extends Cursor<AggregateOperation, ChangeStreamC
   ) {
     super(topology, operation, options);
 
+    this.emitter = new EventEmitter();
```

Comment:
as discussed in team meeting, let's just keep the `EventEmitter` base on the core cursor type